app/ui/src/app/import-export/import/import.actions.ts

```diff
@@ -0,0 +1,117 @@
+import { Action } from '@ngrx/store';
+
+import { ActionReducerError } from '@syndesis/ui/platform';
+import {
+  IntegrationImportRequest,
+  IntegrationUploadRequest,
+} from './import.models';
+import { Integrations} from '@syndesis/ui/model';
+
+export class IntegrationImportActions {
+  static FETCH = '[Import Integration] Fetch integrations request';
+  static FETCH_COMPLETE = '[Import Integration] Fetch operation complete';
+  static FETCH_FAIL = '[Import Integration] Fetch operation failed';
+  static IMPORT = '[Import Integration] Import integration(s) request';
+  static IMPORT_COMPLETE = '[Import Integration] Import integration complete';
+  static IMPORT_FAIL = '[Import Integration] Import integration failed';
+  static UPLOAD = '[Import Integration] Upload integration for import request';
+  static UPLOAD_CANCEL = '[Import Integration] Upload integration for import cancelled';
+  static UPLOAD_COMPLETE = '[Import Integration] Upload integration for import complete';
+  static UPLOAD_FAIL = '[Import Integration] Upload integration for import failed';
+
+
+  /**
+   * Not actually sure that we would need fetching here.
+   */
```

Comment:
It's always better to start with what you really need and then go adding stuff as you need it. Otherwise reducers can become massive and kinda confusing ;)

So, if you see there're actions you're not fully confident that you'll want to use, remove those